go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/moadqassem/operating-system-manager
```

Comment:
This should be:

```suggestion
module k8c.io/operating-system-manager
```

Reply:
Yep, intellij modules isn't familiar with forks 😂 , YET 😉